operations/configuration_management/NCN_Worker_Image_Customization.md

```diff
@@ -0,0 +1,235 @@
+# NCN Worker Image Customization
+
+**NOTE:** Some of the documentation linked from this page mentions use of the Boot Orchestration Service (BOS). The use of BOS
+is only relevant for booting compute nodes and can be ignored when working with NCN images.
+
+This document describes the configuration of a Kubernetes NCN image. The same steps could be used to modify
+a Ceph image.
+
+1. Locate the NCN Image to be Modified
+
+    This example assumes the administrator wants to modify the Kubernetes image that is currently in use by NCNs. However, the steps are the same for any NCN SquashFS image.
+
+    ```console
+    ncn# ARTIFACT_VERSION=<your-version>
+
+    ncn# cray artifacts get ncn-images k8s/$ARTIFACT_VERSION/filesystem.squashfs ./$ARTIFACT_VERSION-filesystem.squashfs
+
+    ncn# cray artifacts get ncn-images k8s/$ARTIFACT_VERSION/kernel ./$ARTIFACT_VERSION-kernel
+
+    ncn# cray artifacts get ncn-images k8s/$ARTIFACT_VERSION/initrd ./$ARTIFACT_VERSION-initrd
+
+    ncn# export IMS_ROOTFS_FILENAME=$ARTIFACT_VERSION-filesystem.squashfs
+
+    ncn# export IMS_KERNEL_FILENAME=$ARTIFACT_VERSION-kernel
+
+    ncn# export IMS_INITRD_FILENAME=$ARTIFACT_VERSION-initrd
+    ```
+
+1. [Import External Image to IMS](../image_management/Import_External_Image_to_IMS.md)
+
+    This document will instruct the administrator to set several environment variables, including the three set in
+    the previous step.
```

Comment:
Should they set them again, because the linked procedure is going to use different values? Or is this being mentioned to say that the duplicate steps can be skipped?

Reply:
The latter.